src/components/user/uc/users.uc.js

```diff
@@ -82,40 +82,30 @@ const replaceUserWithTombstone = async (user) => {
 	return { success: true };
 };
 
-/**
- * Checks if the user defined by param3.account.userId has the right to perform the action defined in permissionAction on the user defined in id
- *
- * @param {*} id ID of the user to be effected by the request
- * @param {*} roleName the role name that the effected user should have
- * @param {*} permissionAction the action that is to be performed (CREATE, EDIT, DELETE)
- * @param {*} param3 needs to contain account.userId, which is the ID of the user issuing the request
- */
-const checkPermissions = async (id, roleName, permissionAction, { user: currentUser }) => {
-	const userToBeEffected = await userRepo.getUserWithRoles(id);
-
+const checkPermissionsInternal = async (affectedUser, roleName, permissionAction, currentUser) => {
```

Comment:
No `await` in the function anymore and therefor no `async` needed.

Reply:
done.